src/Raven.Client/Documents/Operations/OngoingTasks/GetTaskInfoResult.cs

```diff
@@ -299,17 +299,23 @@ public class NextBackup : IDynamicJson
     {
         public TimeSpan TimeSpan { get; set; }
 
-        public DateTime DateTime { get; set; }
+        public DateTime StartTimeLocal { get; set; }
```

Comment:
I don't think we can change it. Operations API is out public API so that would be breaking change. You can add additional property and mark `public DateTime DateTime { get; set; }` as obsolete.

Then both `DateTime ` and `StartTimeLocal ` needs to be provided in `ToJson()`

Reply:
The `DateTime` property name can stay (I changed it for convenience).